api/routes/user.js

```diff
@@ -8,6 +8,23 @@ const SUCCESS = 200;
 const NOT_FOUND = 404;
 const DUPLICATE = 409;
 
+router.get("/", checkToken, (req, res) => {
```

Comment:
We cant have two get endpoints with the same route. Could you remove this one, please?